docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -78,9 +122,8 @@ In this step, we'll set up a new [contact point](https://grafana.com/docs/grafan
 
 1. In your browser, **sign in** to your Grafana Cloud account.
 
-   {{< admonition type="note" >}}
-   **OSS users**: To log in, navigate to localhost:3000, where Grafana is running locally.
-   {{< /admonition >}}
+   **NOTE:**
+   **OSS and Killercoda users**: To log in, navigate to [http://localhost:3000](http://localhost:3000), where Grafana is running locally.
 
 1. In another tab, go to [Webhook.site](https://webhook.site/).
 1. **Copy Your unique URL**.
```

Comment:
Not sure why this step is bold, but it looks odd in the preview.

Reply:
Also, presumably you need to paste this URL somewhere before your webhook would be usable (line 131)?  Feels like that step is missing.  No wait, it's all the way down at step 5 (line 139).